### DISCUSSION TITLE
Outline Not Updating to 0.75.0

### DISCUSSION THREAD

**Discussion by lblue-cmch** [outline/outline#6562] [Self Hosting] [answered]
I have the Business version of outline and I followed the instructions as noted [here](https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t) and it remains on version 0.74.

Is there something I must do differently since I have a license?

**Comment by tommoor** [comment, marked as answer]
Hi Landon – the command is the same, but you need to change the image to the correct one:

`docker pull docker.getoutline.com/outlinewiki/outline-enterprise:latest`

**Reply by lblue-cmch** [reply]
That did it, thank you!
